terraform/shared/config/nomad_client.hcl

```diff
@@ -0,0 +1,28 @@
+data_dir = "/opt/nomad/data"
+bind_addr = "IP_ADDRESS"
+name = "nomad@IP_ADDRESS"
+
+# Enable the client
+client {
+  enabled = true
+  chroot_env {
+    "/bin"              = "/bin"
+    "/etc"              = "/etc"
+    "/home"             = "/home"
```

Comment:
Do we need all of `/home`? If so this is fine, but we might want to explain why in a comment as I wouldn't recommend it for prod deploys. Would hate for it to get copied and pasted by people who don't need it.